docs/feature_oled_driver.md

```diff
@@ -164,6 +164,34 @@ bool oled_task_user(void) {
     return false;
 }
 #endif
+```
+
+Render a message before booting into bootloader mode.
+```c
+void oled_render_boot(void) {
+  oled_clear();
+  for (int i = 0; i < 16; i++) {
+    oled_set_cursor(0, i);
+    oled_write_P(PSTR("BOOT "), false);
+  }
+
+  oled_render_dirty(true);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
```

Comment:
Should use `shutdown_user` instead.

Reply:
oh, I missed that that was a thing .. I'll update ...

Reply:
Yeah, anything that calls `reset_keyboard` or `soft_reset_keyboard` (such as the QK_BOOT keycode) will trigger the shutdown function

Reply:
I see that now, fixed and fixed in my keyboard too. I want to refine it a bit and figure out if I can generalize some of more useful bits before I post that though.